hugo/assets/ts/widgets/tabs.ts

```diff
@@ -56,6 +56,15 @@ export class Tabs extends BaseWidget {
                 }
             }
         });
+
+        // Listen to copybutton events
+        const copyButtons = this.element.querySelectorAll<HTMLElement>('[data-tab-copy]');
```

Comment:
you could also use the copy widget by using data-copy instead of having seperate code for the tabs-copy. 
Would also be great if we can show a message when the item is copied. I think the copy widget already does that.

Reply:
Reverted this change and used the copy widget instead.